lib/resources/Disputes.js

```diff
@@ -8,5 +8,5 @@ module.exports = StripeResource.extend({
 
   includeBasic: ['list', 'retrieve', 'update'],
 
-  close: stripeMethod({method: 'POST', path: '/{id}/close', urlParams: ['id']}),
+  close: stripeMethod({method: 'POST', path: '/{id}/close'}),
```

Comment:
Should we break this out onto multiple lines like we do with most of the `stripeMethod()` calls? Purely a style thing.

Reply:
Good call! prettier collapsed them but we can uncollapse.

Reply:
cc @irace-stripe we may want to try to indicate to prettier-poet that these _should_ break.

Reply:
I think `<Node.Object entries={} break />` would be a useful API, since prettier generally allows objects to be configured to be broken